tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -6,48 +6,51 @@
 import cupy
 from cupy import testing
 from cupyx import fallback_mode
+from cupyx.fallback_mode import ndarray
 
 
-@testing.gpu
-class TestFallbackMode(unittest.TestCase):
+def numpy_fallback_equal(name='xp'):
```

Comment:
Please split this function into `numpy_fallback_equal` and `numpy_fallback_array_equal`: `numpy_fallback_equal` compares return values simply, and `numpy_fallback_array_equal` converts the types of return values.

Reply:
Okay, `numpy_fallback_array_equal` for testing arrays and `numpy_fallback_equal` all other things.

Reply:
Done